pyccel/codegen/printing/ccode.py

```diff
@@ -239,7 +239,7 @@ def __init__(self, parser, settings=None):
         self.known_functions.update(userfuncs)
         self._dereference = set([] if settings is None else settings.get('dereference', []))
         self.prefix_module = prefix_module
-        self._additional_imports = set(['stdlib'])
+        self._additional_imports = set(['stdlib', 'stdint'])
```

Comment:
Is this the right place for this?
@rakati fixed issue #483 with PR #547. I am not sure if this works with his fix or not